background.ts

```diff
@@ -87,6 +89,7 @@ process.on('unhandledRejection', (reason: any, promise: any) => {
 // takes care of any propagation of settings we want to do
 // when settings change
 mainEvents.on('settings-update', async(newSettings) => {
+  console.log(`mainEvents settings-update: ${ JSON.stringify(newSettings) }`);
```

Comment:
Should this be debug?

Reply:
I have no preference either way. It doesn't seem like this would be a problem as far as cluttering up the logs, since settings updates are not that frequent and are generally going to be relevant if someone is trying to solve an issue. Also, it means it isn't as important to that users enable debug mode before sending us logs. I'm happy to change it though if you want.